WMF Framework/Fetcher.swift

```diff
@@ -200,7 +199,7 @@ public class Token: NSObject {
     }
 }
 
-public enum FetcherResult<Success, Error> {
+public enum Result<Success, Error> {
```

Comment:
Should we move this to a separate file and use it everywhere we want a result type?

Reply:
That sounds cool! 😎

Reply:
Can we just use the Swift 5 Result type?

Reply:
good point - forgot that the Result type landed with Swift 5. Will remove this one.